pkg/kubelet/kubelet.go

```diff
@@ -2065,14 +2063,21 @@ func (kl *Kubelet) GetPodStatus(podFullName string, uid types.UID) (api.PodStatu
 }
 
 func (kl *Kubelet) generatePodStatus(podFullName string, uid types.UID) (api.PodStatus, error) {
-	glog.V(3).Infof("Generating status for %s", podFullName)
-
-	spec, found := kl.GetPodByFullName(podFullName)
+	pod, found := kl.GetPodByFullName(podFullName)
 	if !found {
-		return api.PodStatus{}, fmt.Errorf("Couldn't find spec for pod %s", podFullName)
+		return api.PodStatus{}, fmt.Errorf("Couldn't find pod %s", podFullName)
```

Comment:
nit: I know you didn't change it but: errors should start with lowercase and can qe use %q instead of %s? I find it helps in cases when whitespace and empty strings are concerned.